apps/api/src/server/migration/migrators/nsip-exam-timetable-migrator.js

```diff
@@ -4,12 +4,12 @@ import { MigratedEntityIdCeiling } from '../migrator.consts.js';
 import { getCaseIdFromRef, getExamTimetableTypeIdFromName } from './utils.js';
 import * as folderRepository from '#repositories/folder.repository.js';
 import { publish as BroadcastExamTimetable } from '../../applications/examination-timetable-items/examination-timetable-items.service.js';
-import { handleDateTimeToUTC } from '#utils/migration-dates.js';
-
+import { handleDateTimeToUTC, mapDateTimesToCorrectFields } from '#utils/migration-dates.js';
+import { examTimetableItemTypes } from '@pins/examination-timetable-utils';
```

Comment:
I think this is going to need to be added to the api's package.json along with a package-lock update